Library/Formula/ddar.rb

```diff
@@ -8,11 +8,14 @@ class Ddar < Formula
   head "https://github.com/basak/ddar.git"
 
   depends_on "xmltoman" => :build
-  depends_on :python if MacOS.version <= :snow_leopard
-  depends_on "protobuf" => "with-python"
+  depends_on :python
```

Comment:
Anyway, the main issue is that `ddar` depends on python modules installed by itself and its dependencies. If we don't reference `:python`, then it gets installed against the system Python, but the modules are in `/usr/local`. So it will break with a "module not found" error when you actually run it. And if you have the Homebrew Python installed, neither the Caveats nor `brew doctor` will give you the warning about having to add a user `homebrew.pth` to pick up those modules, and you'll get the issue jiegec was discussing [here](https://github.com/Homebrew/homebrew/pull/44621#discussion_r41247046).